src/components/Reactions/QuickEmojiReactions/types.ts

```diff
@@ -0,0 +1,66 @@
+import {TextInput} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import type {Emoji} from '@assets/emojis/types';
+import type {AnchorOrigin} from '@userActions/EmojiPickerAction';
+import type {Locale, ReportAction, ReportActionReactions} from '@src/types/onyx';
+
+// TODO: remove when ReportActionContextMenu file migration https://github.com/Expensify/App/pull/32670 is merged
```

Comment:
```suggestion
// TODO: Remove this once AnchorForCommentsOnly (https://github.com/Expensify/App/issues/25135) is migrated to TypeScript.
```

To be better aligned with the docs.